docs/content/manual/manual.yml

```diff
@@ -3344,6 +3351,23 @@ sections:
 
           Returns the line number of the input currently being filtered.
 
+      - title: "`exec(path)`, `exec(path; [args…])`"
+        body: |
+
+          Spawns a new process of path with arguments args. Pipes its input
+          converted to a string as the stdin of the process. Outputs an object
+          like `{'out': "", "err": "", "status": 0}`, containing the stdout,
+          stderr, and exit code of the process respectively. If the process
+          exits abnormally due to an unhandled signal, it will have a status of
+          "-1" and a "signal" parameter giving the numeric value of the signal
+          that caused the process to terminate.
+
+      - title: "`system(path)`, `system(path; [args…])`"
```

Comment:
FWIW, in most other languages, the difference between `exec` and `system` is usually that `system` invokes `$SHELL -c` (and thus would normally only take a string argument).

(https://pubs.opengroup.org/onlinepubs/9699919799/functions/system.html)

I personally don't think it matters that much, but it is probably worth either starting with _just_ `exec` or naming this function something else.

(And to be really clear, I'm not associated with the project at all; just a heavy user who wants `exec` :smile: :heart:)

Reply:
The PR's been there for a year, I'm pretty sure the project isn't interested in having this merged.
I really do not think the naming's the issue here. :)
Realistically, I should just close it.